docs/source/pillars/information_governance.md

```diff
@@ -91,6 +91,14 @@ _The ability of the TRE organisation to monitor compliance with internal and ext
 * - You should report on and share outcomes of each audit of your TRE organisation with the required bodies.
   - This may be a requirement of continued accreditation by external organisations or regulatory bodies.
   - Recommended
+* - You log data access queries and publish these logs
```

Comment:
Echoing what I said before,

> Publishing logs to prove "we have nothing to hide" feels virtuous. I'm not sure how much use logs are to the public though or what volume we are talking about. Do we believe members of the public are going to be enabled to, or have the enthusiasm to analyse our logs?

The way this is written, you fail if you record the logs but do not publish them.
They should at least be separate components.

Reply:
Idk, it's one of those heuristic things where even if you don't understand it, knowing the org is doing it publicly so someone _can_ interrogate it if they want to I think is good

Reply:
Definitely agree that these are two separate points. Unclear what level of logging is necessary here. Is saying "Jane Doe was given access to this data between 01/01/2023 and 31/03/2023" sufficient? Are we requiring something like "2023-02-17T16:09:34 Jane Doe opened file `data/missile_blueprints/final_fixed_v2/hatch4/TODO_FIX.png`"?

Reply:
Came here to mention that - I'd also be slightly concerned about the potential for logs to leak sensitive/personal info, both about the nature of the data and potentially things like which person is working on which data. Might sound innocuous, but imagine one project is highly commercially sensitive, or some kind of defence project...

Reply:
I don't think publishing routine logs should be recommended. As mentioned by others there's a high risk of sensitive information being leaked. Logs originate from inside the TRE, so publishing logs effectively means data that originates from inside the TRE is being automatically egressed.

The Goldacre report says
> TRE 16. All TREs must share live detailed activity logs

but my interpretation of this is "logs" means a "record" of activity, and not operational logging. To get this information safely you'd need to have structured logging or audit events, that are subsequently processed so very limited information is published.

Reply:
@manics I think I would prefer something along those lines as a recommendation or optional component. It sounds like implementing that would require a lot of technical effort up front as well as continuous oversight of what is released.

It would be quite complementary to the [PIE component](https://github.com/sa-tre/satre-specification/blob/392404cfc41ff410c33c01798a69378f2c08c521/docs/source/pillars/supporting.md?plain=1#L177) about publishing details of projects publicly. I think the cases where you would want to share 'activity logs' and project information are similar.